.eslintrc.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
     "react/prop-types": "off",
     // this is the default in typescript and we want to enforce it in JavaScript as well
     "prefer-const": "error",
+    "no-nested-ternary": "off"
```

Comment:
This is very important, please never deactivate an eslintrule without communication. I'm happy to discuss any rule that is listed here.

Reply:
Yeah, I will make it sure